api/python/ai/chronon/repo/hub_uploader.py

```
@@ -48,10 +54,16 @@ def _build_local_repo_hashmap(root_dir: str):
                 name = json_obj["metaData"]["name"]
 
                 # Load the json into the appropriate object type based on folder
-                binary = json2binary(thrift_json, obj_class)
+                # binary = json2binary(thrift_json, obj_class)
 
                 md5_hash = hashlib.md5(thrift_json.encode()).hexdigest()
-                results[name] = (binary, md5_hash)
+                # md5_hash = hashlib.md5(thrift_json.encode()).hexdigest() + "123"
+                # results[name] = (binary, md5_hash, FOLDER_NAME_TO_LOGICAL_TYPE[folder_name])
+                results[name] = Conf(name=name,
+                                          hash=md5_hash,
```

Flake8 (7.2.0)

[error] continuation line over-indented for visual indent

(E127)